lblod/mandataris-service
# Refactor burgemeesterbenoeming

## Description

Split burgemeesterbenoeming in multiple files (controller, data-access ...) and merged some logic. This is just a refactor, the logic is still the same.

## How to test

Check if the burgemeesterbenoemingen still works.

Can be done with the following collection:
- bruno: [burgemeester-benoemingen-bruno.json](https://github.com/user-attachments/files/16581785/burgemeester-benoemingen-bruno.json)
- postman: [burgemeester-benoemingen-postman.json](https://github.com/user-attachments/files/16581790/burgemeester-benoemingen-postman.json)

The get file should also be tested, I couldn't get it to work, but also not on master, so not sure I'm using the correct id. The call to get a file should look like this: 
http://localhost:90/files/77971b9f-2b87-4135-b254-db2237b036a1/download

## Linked PR's
- refactor of: https://github.com/lblod/mandataris-service/pull/2
